### PR TITLE
refactor: cleanup flagsmith usage and set default values

### PR DESCRIPTION
**Pull request by sshanzel**
Cleanup of Flagsmith for the API side.

- Hard-coded values for the following flags:
  - Onboarding v2: always at `v1` then the existing logic to update it as necessary is still in place.
  - Feed version: value is now always `11`.
  - Submit article: we have an existing mechanism that updates the flags to include it when the user is eligible.
    - We will keep the same process to support outdated extension users.
    - We will introduce an additional property to tag whether the user is eligible (though we already have a query that returns the same value, the `query submissionAvailability` - maybe we should remove it from there).
- We should probably run a local script to clear off our cache from flagsmith values. Thoughts?

### CODE REVIEW

File: .infra/workers.ts
Comment (idoshamun) [inline .infra/workers.ts:203]: We can delete this topic from `streams` once it's merged

**Review by idoshamun — APPROVED** [review]
Looks great!! We can either let the cache expire or delete manually. Not sure we need a script for that

**Comment by sshanzel** [comment]
Good point! Forgot they actually expire on their own. In that case, we won't have to run anything.

**Review by rebelchris — APPROVED** [review]
Cleanup! 🧼 

Looks pretty complete, think you also have the overwrites ok to work backwords compatible 🙏

**Comment by sshanzel** [comment]
> Cleanup! 🧼
> 
> Looks pretty complete, think you also have the overwrites ok to work backwords compatible 🙏

Yep, I will merge them right after one another. We might have some fluctuations with the distribution since extension users will always be on default.

**Pull request merged**
